python/ml4ir/features/feature_layer.py

```diff
@@ -0,0 +1,107 @@
+import tensorflow as tf
+from tensorflow import feature_column
+from tensorflow.keras import layers
+from tensorflow.keras import backend as K
+
+from ml4ir.features.feature_config import FeatureConfig
+from ml4ir.config.keys import FeatureTypeKey, EncodingTypeKey, TFRecordTypeKey
+
+
+def get_dense_feature(inputs, feature, shape=(1,)):
+    """
+    Convert an input into a dense numeric feature
+
+    NOTE: Can remove this in the future and
+          pass inputs[feature] directly
+    """
+    feature_col = feature_column.numeric_column(feature, shape=shape)
+    dense_feature = layers.DenseFeatures(feature_col)(inputs)
+
+    return dense_feature
+
+
+def get_sequence_embedding(input, feature_info, max_num_records):
```

Comment:
`get_sequence_encoding`?  since not always an embedding

Reply:
Will change.